packages/models/src/lib/upload-config.ts

```diff
@@ -9,6 +9,11 @@ export const uploadConfigSchema = z.object({
   }),
   organization: slugSchema('Organization slug from Code PushUp portal'),
   project: slugSchema('Project slug from Code PushUp portal'),
+  timeout: z
+    .number({ description: 'Request timeout in minutes (default is 5)' })
```

Comment:
Zod supports a default value. Or is it cleaner to only have it set on `portal-client`?

Reply:
Yeah, I didn't want to have it set in two places.